compiler/one-cmds/one-quantize

```diff
@@ -120,6 +132,7 @@ echo " " >> "${OUTPUT_PATH}.log"
 "${DRIVER_PATH}/record-minmax" \
 --input_model "${TMPDIR}/${MODEL_NAME}.1.circle" \
 --input_data "${INPUT_DATA}" \
+--min_percentile ${MIN_PERCENTILE} --max_percentile ${MAX_PERCENTILE} \
```

Comment:
Maybe "${MIN_PERCENTILE}" and "${MAX_PERCENTILE}" ? (added " ")

Reply:
It's number and one word (without space) so didn't add `" "`. Its OK.
`quantize_with_minmax ` is passed with `--quantize_with_minmax ${INPUT_DTYPE} ${QUANTIZED_DTYPE} ${GRANULARITY}`